packages/shared/src/utils/aggregation.ts

```diff
@@ -111,3 +111,35 @@ export function aggregateDaily(data: AggregatedData[], start: Date, end: Date):
 
   return Array.from(dailyMap.values());
 }
+
+/**
+ * 指定した期間内のデータを時間単位で集計
+ */
+export function aggregateHourly(data: AggregatedData[]): AggregatedData[] {
+  const hourlyMap = new Map<string, AggregatedData>();
+  data.forEach((row) => {
+    const date = new Date(row[AGGREGATE_FROM_KEY]);
+    const hourKey = `${date.getFullYear()}-${String(date.getMonth() + 1).padStart(2, "0")}-${String(date.getDate()).padStart(2, "0")} ${String(date.getHours()).padStart(2, "0")}:00`;
+
+    const dayOfWeek = ["日", "月", "火", "水", "木", "金", "土"][date.getDay()];
```

Comment:
```suggestion
    const dayOfWeek = WEEK_DAYS[date.getDay()];
```

同様に手元で反映してもらいたいですが、このようにするべきだと思います。